src/server/scriptInfo.ts

```diff
@@ -559,10 +543,15 @@ namespace ts.server {
             }
         }
 
-        getLatestVersion(): string {
+        getLatestVersion() {
             // Ensure we have updated snapshot to give back latest version
-            this.textStorage.getSnapshot();
-            return this.textStorage.getVersion();
+            const snapShot = this.textStorage.getSnapshot();
+            if (this.textStorage.version === undefined) {
+                // Ensure we have updated snapshot to give back latest version
+                const text = getSnapshotText(snapShot);
+                this.textStorage.version = this.host.createHash ? this.host.createHash(text) : generateDjb2Hash(text);

```

Comment:
All of our implementations will use SHA256, right?  Djb2 is just there as a fallback for other consumers?

Reply:
Yes